### ISSUE TITLE
Email icon isn't visible in dark mode

### ISSUE THREAD

**Issue by tutaf** [medyo/android-about-page#153]
Email icon is almost the same color as background, so it's not visible in dark mode. It would be good to change the color of icon according to the dark mode.

**Comment by Hussienfahmy** [comment]
GitHub icon Also

**Comment by MrReSc** [comment]
Website icon is also not visible.

**Comment by medyo** [comment]
Can you share a screenshot ?

**Comment by Hussienfahmy** [comment]
![Screenrecorder-2021-03-07-14-18-25-853.gif](https://user-images.githubusercontent.com/65970301/110239751-41ec0f00-7f51-11eb-99af-9fe5f445e02f.gif)

**Comment by nanduk667** [comment]
Any solution for this yet?

**Comment by tutaf** [comment]
> Any solution for this yet?

You may try to make a custom item and change icon color according to dark mode setting

**Comment by nanduk667** [comment]
> > Any solution for this yet?
> 
> You may try to make a custom item and change icon color according to dark mode setting

yeah, doing that currently. But for things like email, this was much simpler.
